.env.example

```diff
@@ -0,0 +1,3 @@
+#Example of .env file. Fill below secrets for local development.
+#INFURA_PROJECT_ID=
+#INFURA_PROJECT_SECRET=
```

Comment:
Could you update the README with instructions of what one should do with `.env.example` in order to run Switchboard locally please?

Reply:
Should i add link to the confluence page? Or just information that user needs to provide those values?

Reply:
Hmm, good question. I think the instructions can assume that the person has Infura credentials. I think we need some steps like:
1. Execute `cp .env.example .env`
2. Fill in necessary variables
Does that make sense from your perspective?

Reply:
Yup, added to the readme. Hope the place where i added it is good in you opinion.

Reply:
Yes, looks good, thanks!